docs/python/index.md

```diff
@@ -2,16 +2,6 @@
 To provide greater flexibility and support use-cases that are difficult to express using the CLI
 (such as dependencies between tasks), HyperQueue also offers a Python API for submitting complex jobs.
 
-There are several benefits to using the Python API, which are not easily available in the command-line
-interface:
-
-- [**Task dependencies**](dependencies.md) You can specify dependencies between individual
```

Comment:
Well dependencies are still not in jobfile, right? :) Same as with Python tasks.